### ISSUE TITLE
screenplay for podcast

### ISSUE THREAD

**Issue by SanliFaez** [SanliFaez/Commons4Academics#15]
Thoughts from Andy Clark, our producer

How it can work: 

I think that it is good to collect some quotes from participants at the start of the day to get their expectations and sense of anticipation … then we need to set up what the main issues are and back these up to with good quotes from the key speeches. 

I would also ideally interview the key speakers - a brief interview of 10/15  minutes - so I will need a room that I can take people too and interview and of course it’s best that everyone knows I will be grabbing them at some stage. 

I will record sound from the breakaway groups and interview at least one spokesperson from a group to see what they have been working  on and what they came up with as a solution. This will serve as an example of one of the steps that can ben taken.

I would need to interview one fo the main players at the end of the day again to get a round up of the new ideas and take aways and next steps. Good to identify this person ahed of time and fix this up. 

I will also collect quotes from participants at the end of the day too to see what they have learned and hear their passion about implementing these new ideas. 

I am thinking about a main podcast of 40-45 minutes.

Alongside this I can record all the keynotes if you want me too so you can make those available. And separate interviews with key players can also be published in full if you want.

The bottom line is that the podcast really needs to emphasize why this is necessary, the gains that can be made  and the way ahead to achieving this. 

Let’s fix up a time to skype on this or maybe we can meet up in Utrecht ahead of time to fix things up in more detail - you can give me a briefing on the key speakers for example and what I can best ask them.

**Comment by JeroenBosman** [comment]
Sounds fine to me, though I would try to get it back to 30 mins in the editing phase. Especially as this serves to reach a broader group, 45 mins may be a bit too long?
